code/game/machinery/Sleeper.dm

```diff
@@ -449,25 +453,23 @@
 /obj/machinery/sleeper/ex_act(severity)
 	if(filtering)
 		toggle_filter()
+	var/mob/M = src.occupant

```

Comment:
What @tigercat2000 means is that you don't need to put src.occupant - just occupant will do as src is implied